framework/src/main/java/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -538,6 +544,29 @@ private Range calculateLengthRangeAddition(Range leftLengths, Range rightLengths
         return leftLengths.plus(rightLengths).intersect(Range.INT_EVERYTHING);
     }
 
+    /**
+     * This method is only used to check for nullable nodes in string concatenations.
+     *
+     * @return true if the node can't be null or false otherwise.
```

Comment:
I don't think this comment is quite correct, because this method returns "true" for String concatenation nodes, which are definitely nullable in general. Can you be more precise in the description (and the name of the method)?